src/pages/index.tsx

```diff
@@ -40,6 +42,10 @@ export default function HomePage() {
       <Navigation activeTab={NavigationTab.Home} />
       <Container>
         <MainBanner />
+      </Container>
+      <MetricsCards />
```

Comment:
Is this container really necessary? I think it would be cleaner if we just use a single container that contains all the components of the home page

Reply:
We need to render the Container conditionally in this component as it is not used in mobile.
<img width="450" alt="image" src="https://user-images.githubusercontent.com/8242162/183657679-44bdbdda-3ee3-4518-8cf8-1c37c45776c5.png">